rtsp.go

```diff
@@ -38,6 +42,7 @@ var ModelH264 = family.WithModel("rtsp-h264")
 var ModelH265 = family.WithModel("rtsp-h265")
 var ModelMJPEG = family.WithModel("rtsp-mjpeg")
 var Models = []resource.Model{ModelAgnostic, ModelH264, ModelH265, ModelMJPEG}
+var ErrH264PassthroughNotEnabled = errors.New("H264 passthrough is not enabled")
```

Comment:
[nit] Not a gorista, but should we format in a var ( ) block here?

```
var (
    ModelH265                   = family.WithModel("rtsp-h265")
    ModelMJPEG                  = family.WithModel("rtsp-mjpeg")
    Models                      = []resource.Model{ModelAgnostic, ModelH264, ModelH265, ModelMJPEG}
    ErrH264PassthroughNotEnabled = errors.New("H264 passthrough is not enabled")
)
```

Reply:
good call, done